lib/CIndexStoreDB/CIndexStoreDB.cpp

```diff
@@ -144,6 +144,69 @@ indexstoredb_symbol_name(indexstoredb_symbol_t symbol) {
   return obj->value->getName().c_str();
 }
 
+/// loops through each symbol in the index and calls the receiver function with each symbol
+/// @param index an IndexStoreDB object which contains the symbols
+/// @param receiver a function to be called for each symbol, the CString of the symbol will be passed in to this function.
+/// The function should return a boolean indicating whether the looping should continue.
+bool
```

Comment:
The documentation comments should go in the header, not the .cpp file.  Please capitalize the first word as well.  Same for the other functions.